nyc_taxis/test_procedures/default.json

```diff
@@ -122,6 +122,34 @@
           {%-if search_clients is defined and search_clients %}
           ,"clients": {{ search_clients | tojson}}
           {%- endif %}
+        },
+        {
+          "operation": "desc_sort_tip_amount",
+          "warmup-iterations": 200,
```

Comment:
Does it need 200 warmup iterations?

Reply:
Yeah, that is too high ! Made it 10. `10` is enough for sort queries.